### ISSUE TITLE
Second const unnecessary and in fact wrong

### ISSUE THREAD

**Issue by RealWeeks** [ga-wdi-boston/js-objects-constructors#24]
``` js
const foo = 'bar';
const foo = 'baz'; // wrong!
```

Think intended:

``` js
const foo = 'bar';
foo = 'baz'; // Explode
```

via @gaand

**Comment by jrhorn424** [comment]
Addressed in 05b73504476e56d633a2619e21bb3ab09a1709a0

**Comment by gaand** [comment]
I think I intended to just show:

``` js
const foo = 'bar';
foo = 'baz'; // Explode
```

That doesn't mean having both is a problem but I'm taking it out to limit cognitive load.
